Wikipedia/Code/WMFWelcomePanelViewController.swift

```diff
@@ -0,0 +1,58 @@
+
+class WMFWelcomePanelViewController: UIViewController {
```

Comment:
Can you think of a better name for this? Its not really clear what it does from the name